.github/workflows/foreman_plugin.yml

```
@@ -166,6 +166,11 @@ jobs:
         if: ${{ hashFiles(format('{0}/package.json', inputs.plugin), format('{0}/test/integration/**', inputs.plugin)) != '' && contains(matrix.task, 'test') }}
         run:
           bundle exec rake webpack:compile
+      - name: Archive all_react_app_exports
+        uses: actions/upload-artifact@v4
+        with:
+          name: all_react_app_exports-${{ env.ARTIFACT_SUFFIX }}.js
+          path: foreman/webpack/assets/javascripts/all_react_app_exports.js
```

```suggestion
          path: webpack/assets/javascripts/all_react_app_exports.js
```

We do not deploy foreman in a `foreman` subfolder.

---

Changed, thanks